less/home-page.less

```
@@ -1,9 +1,14 @@
 // Home Content
 
 .home {
-
+  padding-top: 100px;
   .intro {
-
+    img {
+      max-width: 800px;
```

```suggestion
      max-width: 800px;
      width: 100%;
```
This makes it behave just a little bit better